hw_1/task_2.py

```diff
@@ -0,0 +1,12 @@
+a = int(input('Введите сторону треугольника '))
+b = int(input('Введите сторону треугольника '))
+c = int(input('Введите сторону треугольника '))
+
+if a + b < c or a + c < b or b + c < a:
+    print('треугольника с такими сторонами не существует')
+elif a == b == c:
+    print('треугольник равносторонний')
+elif a == b != c or a == c != b or b == c != a:
```

Comment:
У вас тут исключаемое условие
```suggestion
elif a == b or a == c or b == c:
```